modules/sdk/gradle-plugins-node/src/main/java/com/liferay/gradle/plugins/node/NodeExtension.java

```diff
@@ -340,9 +400,11 @@ public void setYarnVersion(Object yarnVersion) {
 	private final List<Object> _npmArgs = new ArrayList<>();
 	private Object _npmUrl;
 	private Object _npmVersion;
+	private final Object _pnpmUrl;
+	private final Object _pnpmVersion = "8.6.10";
 	private final Project _project;
 	private Object _scriptFile;
-	private Object _useNpm = true;
+	private Object _usingNPM = "yarn";
```

Comment:
Why is the default changing to `yarn` here?

Reply:
Because we are using yarn as default package manager in liferay workspace now (https://github.com/liferay/liferay-portal/blob/e2734632486accdb4ef0ea2df712365c77e85932/modules/sdk/gradle-plugins-node/src/main/java/com/liferay/gradle/plugins/node/NodePlugin.java#L435), so I think if there is no requirement to change it, we should keep it as the default package manager.